### ISSUE TITLE
Return to UUID page instead of DB rowID after link/unlink

### ISSUE THREAD

**Issue by Hawke** [acoustid/acoustid-server#24]
When you unlink or link an AcoustID from a MusicBrainz recording ID, the page you are returned to references what I assume to be the database rowID. It should instead go to the UUID.

IMO the rowID should never be exposed, and e.g. http://acoustid.org/track/12534071 should return an error — http://acoustid.org/track/12ec629c-dd1c-422a-9e3b-90f12280ea35 should be the only way to reference that AcoustID.

**Comment by jesus2099** [comment]
I agree, it’s very inconvenient because we then copy some uninformative URLs back to edit notes or MusicBrainz forum.
